app/views/articles/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @article.title %></h1>
+
+<p><%= @article.body %></p>
+
+<ul>
+  <li><%= link_to "Edit", edit_article_path(@article) %></li>
+  <li><%= link_to "Destroy", article_path(@article),
+                  method: :delete,
```

Comment:
@sansuaza what is the purpose of `method: :delete` ?

Reply:
indicates the method by which the request will be executed

Reply:
@sansuaza this is not a correct answer. Please research about it

Reply:
The delete method is indicated with the porpuse that the binding not make a get request

Reply:
@sansuaza good!